bucket/static-web-server.json

```diff
@@ -0,0 +1,50 @@
+{
+    "version": "2.9.0",
+    "description": "A blazing fast and asynchronous web server for static files-serving",
+    "homepage": "https://sws.joseluisq.net/",
+    "license": "MIT|Apache-2.0",
+    "notes": [
+        "To run the server as a Windows Service have a look at https://sws.joseluisq.net/features/windows-service"
+    ],
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/joseluisq/static-web-server/releases/download/v2.9.0/static-web-server-v2.9.0-x86_64-pc-windows-msvc.zip",
+            "hash": "72e846329e54c0a794e5c2c62484266f31f0c411bf4243a1bac58e7c9d697d3d"
+        },
+        "32bit": {
+            "url": "https://github.com/joseluisq/static-web-server/releases/download/v2.9.0/static-web-server-v2.9.0-i686-pc-windows-msvc.zip",
+            "hash": "8f3cd9c447136b7da200099b2265e311085debb9f2c2fc30ff8bf732f6ec312c"
+        }
+    },
+    "bin": [
+        "static-web-server.exe",
+        [
+            "static-web-server.exe",
+            "static-web-server"
+        ],
+        [
+            "static-web-server.exe",
+            "sws.exe"
+        ],
```

Comment:
```suggestion
```
This is not needed, because it does the same thing as `static-web-server.exe` -> `sws`